### PR TITLE
Using eslint 

### PR DESCRIPTION
**Pull request by elahehrashedi**
as a part of refactoring package.json, I applied this change as well

### CODE REVIEW

File: src/cmake-tools.ts [hunk 1]
```diff
@@ -1387,15 +1389,16 @@ export class CMakeTools implements vscode.Disposable, api.CMakeToolsAPI {
...
+  }
 
   /**
    * Implementation of `cmake.cleanRebuild`
    */
   async cleanRebuild(): Promise<number> {
     const clean_res = await this.clean();
-    if (clean_res !== 0)
-      return clean_res;
+    if (clean_res !== 0) {return clean_res; }
```
Comment (bobbrow) [inline src/cmake-tools.ts:1401]: wrap this line

File: src/drivers/cms-driver.ts [hunk 2]
```diff
@@ -256,12 +255,10 @@ export class CMakeServerClientDriver extends codemodel.CodeModelDriver {
 
   get cmakeCacheEntries(): Map<string, CacheEntryProperties> { return this._cacheEntries; }
 
-
   private async _setKitAndRestart(need_clean: boolean, cb: () => Promise<void>) {
     this._cmakeInputFileSet = InputFileSet.createEmpty();
     const client = await this._cmsClient;
-    if (client)
-      await client.shutdown();
+    if (client) {await client.shutdown(); }
```
Comment (bobbrow) [inline src/drivers/cms-driver.ts:261]: wrap this line

File: src/drivers/driver.ts [hunk 3]
```diff
@@ -467,8 +466,7 @@ export abstract class CMakeDriver implements vscode.Disposable {
     log.debug(localize('cmakedriver.kit.set.to', 'CMakeDriver Kit set to {0}', kit.name));
     this._kitEnvironmentVariables = await effectiveKitEnvironment(kit, this.expansionOptions);
 
-    if (kit.preferredGenerator)
-      preferredGenerators.push(kit.preferredGenerator);
+    if (kit.preferredGenerator) {preferredGenerators.push(kit.preferredGenerator); }
```
Comment (bobbrow) [inline src/drivers/driver.ts:469]: wrap this line

File: src/drivers/driver.ts [hunk 4]
```diff
@@ -1491,8 +1487,7 @@ export abstract class CMakeDriver implements vscode.Disposable {
       await child.result;
       this._currentBuildProcess = null;
       return child;
-    } else
-      return null;
+    } else {return null; }
```
Comment (bobbrow) [inline src/drivers/driver.ts:1490]: wrap this line

File: src/drivers/driver.ts [hunk 5]
```diff
@@ -1510,8 +1505,7 @@ export abstract class CMakeDriver implements vscode.Disposable {
 
     const cur = this._currentBuildProcess;
     if (cur) {
-      if (cur.child)
-        await util.termProc(cur.child);
+      if (cur.child) {await util.termProc(cur.child); }
```
Comment (bobbrow) [inline src/drivers/driver.ts:1508]: wrap this line

File: src/extension.ts [hunk 6]
```diff
@@ -1166,7 +1164,7 @@ class ExtensionManager implements vscode.Disposable {
   }
 
   async selectWorkspace(folder?: vscode.WorkspaceFolder) {
-    if (!folder) return;
+    if (!folder) {return; }
```
Comment (bobbrow) [inline src/extension.ts:1167]: wrap this line

Comment (bobbrow) [inline src/extension.ts:1167]: wrap this line

File: src/kit.ts [hunk 7]
```diff
@@ -284,8 +284,7 @@ export async function kitIfCompiler(bin: string, pr?: ProgressReporter): Promise
   const clang_res = clang_regex.exec(fname);
   if (gcc_res) {
     log.debug(localize('testing.gcc.binary', 'Testing GCC binary: {0}', bin));
-    if (pr)
-      pr.report({message: localize('getting.gcc.version', 'Getting GCC version for {0}', bin)});
+    if (pr) {pr.report({message: localize('getting.gcc.version', 'Getting GCC version for {0}', bin)}); }
```
Comment (bobbrow) [inline src/kit.ts:287]: wrap this line

File: src/kit.ts [hunk 8]
```diff
@@ -342,8 +340,7 @@ export async function kitIfCompiler(bin: string, pr?: ProgressReporter): Promise
 
   } else if (clang_res) {
     log.debug(localize('testing.clang.binary', 'Testing Clang binary: {0}', bin));
-    if (pr)
-      pr.report({message: localize('getting.clang.version', 'Getting Clang version for {0}', bin)});
+    if (pr) {pr.report({message: localize('getting.clang.version', 'Getting Clang version for {0}', bin)}); }
```
Comment (bobbrow) [inline src/kit.ts:343]: wrap this line

File: src/kit.ts [hunk 9]
```diff
@@ -323,8 +322,7 @@ export async function kitIfCompiler(bin: string, pr?: ProgressReporter): Promise
           log.debug(localize('bad.mingw32-make.binary', 'Bad mingw32-make binary ("-v" returns non-zero): {0}', bin));
         } else {
           let make_version_output = execMake.stdout;
-          if (make_version_output.length === 0)
-            make_version_output = execMake.stderr;
+          if (make_version_output.length === 0) {make_version_output = execMake.stderr; }
```
Comment (bobbrow) [inline src/kit.ts:325]: wrap this line

File: src/pr.ts [hunk 10]
```diff
@@ -94,9 +94,9 @@ export function copyFile(inpath: string, outpath: string): Promise<void> {
     reader.on('error', e => reject(e));
     reader.on('open', _fd => {
       const writer = fs_.createWriteStream(outpath);
-      writer.on('error', e => reject(e));
+      writer.on('error', reject);
```
Comment (bobbrow) [inline src/pr.ts:97]: revert

File: src/pr.ts [hunk 11]
```diff
@@ -94,9 +94,9 @@ export function copyFile(inpath: string, outpath: string): Promise<void> {
...
       writer.on('open', _fd2 => { reader.pipe(writer); });
-      writer.on('close', () => resolve());
+      writer.on('close', resolve);
```
Comment (bobbrow) [inline src/pr.ts:99]: revert

File: src/proc.ts [hunk 12]
```diff
@@ -87,7 +89,7 @@ export interface ExecutionOptions {
 
 export function buildCmdStr(command: string, args?: string[]): string {
   let cmdarr = [command];
-  if (args) cmdarr = cmdarr.concat(args);
+  if (args) {cmdarr = cmdarr.concat(args); }
```
Comment (bobbrow) [inline src/proc.ts:92]: wrap this line

File: src/proc.ts [hunk 13]
```diff
@@ -145,18 +146,15 @@ export function execute(command: string,
 
     result = new Promise<ExecutionResult>((resolve, reject) => {
       resolve({retc: 0, stdout: '', stderr: ''});
-      if (false) reject();
+      if (false) {reject(); }
```
Comment (bobbrow) [inline src/proc.ts:149]: wrap this line

File: src/proc.ts [hunk 14]
```diff
@@ -166,8 +164,7 @@ export function execute(command: string,
         })
       };
     }
-    if (options.encoding)
-      child.stdout?.setEncoding(options.encoding);
+    if (options.encoding) {child.stdout?.setEncoding(options.encoding); }
```
Comment (bobbrow) [inline src/proc.ts:167]: wrap this line

File: src/tree.ts [hunk 15]
```diff
@@ -550,8 +550,8 @@ export class ProjectOutlineProvider implements vscode.TreeDataProvider<BaseNode>
     }
   }
 
-  setActiveFolder(ws: vscode.WorkspaceFolder | undefined):void {
-    if (!ws) return;
+  setActiveFolder(ws: vscode.WorkspaceFolder | undefined): void {
+    if (!ws) {return; }
```
Comment (bobbrow) [inline src/tree.ts:554]: wrap this line

File: src/util.ts [hunk 16]
```diff
@@ -286,8 +283,7 @@ async function _killTree(pid: number) {
       children = stdout.split('\n').map(line => Number.parseInt(line));
     }
     for (const other of children) {
-      if (other)
-        await _killTree(other);
+      if (other) {await _killTree(other); }
```
Comment (bobbrow) [inline src/util.ts:286]: wrap this line

Comment (bobbrow) [inline src/util.ts:286]: wrap this line

**Comment by elahehrashedi** [comment]
@bobbrow  I added a rule that can be applied to auto fix all of those wrapping issues, but I commented it out for now, because it can only autofix when indentation is also working (applying it alone doesn't use the correct indentation). The rule can stay there and be applied later (I commented on the code).

**Review by bobbrow — APPROVED** [review]

**Pull request merged**